module.json

```diff
@@ -2,76 +2,109 @@
   "id": "morkborg-3p",
   "title": "MÖRK BORG Third-Party Content",
   "description": "Third-party content for the MÖRK BORG system.",
-  "version": "This is auto replaced",
-  "library": "false",
-  "manifestPlusVersion": "1.0.0",
+  "version": "2.2.0",
   "compatibility": {
-    "minimum": 11,
-    "verified": 11
-  },   
+    "minimum": "11",
+    "verified": "11.300"
+  },
   "authors": [
     {
-      "name": "mcglintlock"
-    }    
-  ],  
+      "name": "mcglintlock",
+      "flags": {}
+    }
+  ],
   "relationships": {
-    "systems": [{
-      "id": "morkborg",
-      "type": "system",
-      "manifest": "https://github.com/fvtt-fria-ligan/morkborg-foundry-vtt/releases/download/2.0.3/system.json",
-      "compatibility": {
-        "verified": "2"
+    "systems": [
+      {
+        "id": "morkborg",
+        "type": "system",
+        "manifest": "https://github.com/fvtt-fria-ligan/morkborg-foundry-vtt/releases/download/2.0.3/system.json",
+        "compatibility": {
+          "verified": "2"
+        }
       }
-    }]
-  },   
-  "esmodules": ["/scripts/module.js"],
+    ]
+  },
+  "esmodules": [
+    "/scripts/module.js"
+  ],
   "packs": [
     {
       "name": "mork-borg-3p-actors",
       "label": "MÖRK BORG 3P (Actors)",
+      "path": "packs/mork-borg-3p-actors",
+      "type": "Actor",
       "system": "morkborg",
-      "path": "./packs/mork-borg-3p-actors",
-      "type": "Actor"
+      "ownership": {
```

Comment:
Is this ownership addition actually needed? where did it come from?

Reply:
I think this might be a typo error, good catch.